resources/views/account/accept/create.blade.php

```diff
@@ -94,6 +103,24 @@
 @section('moar_scripts')
 
     <script nonce="{{ csrf_token() }}">
+        $(document).ready(function(){
+
+            $('#note').hide();
+            $('#note_label').hide();
+
+
+            $('input[id="declined"]').change(function(){
+
+                if($(this).is(':checked')){
+                    $('#note_label').show();
+                    $('#note').show();
+                }
+                else {
+                    $('#note_label').hide();
+                    $('#note').hide();
+                }
+            });
+        });
```

Comment:
I think we can skip the hiding of the notes field and simply always have it displayed like the radio buttons.

Reply:
The textarea doesn't appear until the `I Decline` option is selected. We should have that pop up when either is selected (if we hide by default)

![CleanShot 2024-03-21 at 14 45 16](https://github.com/snipe/snipe-it/assets/1141514/af94601b-70b1-4a86-b615-f275c6008ad6)